README.md

```diff
@@ -220,6 +218,42 @@ component to use for rendering the component. See the example below.
 <RenderCmsComponent path={'menu'} renderComponent={CmsMenu} />
 ```
 
+### Container components
```

Comment:
Let's stick with the term 'containers' instead of 'container components'.

Reply:
The same applies to line 223

Reply:
Fixed, thanks 👍